draft-ietf-quic-multipath.md

```diff
@@ -132,18 +132,24 @@ and the actual decision to setup or tear down paths are assumed
 to be handled by the application. But this document does not prevent future extensions from
 defining mechanisms to cope with the remaining scenarios.
 
-Further, this document does not specify detailed algorithms that define
+Further, this document does not specify scheduling algorithms that define
 how multiple, simultaneously open paths are used to send packets.
-As these differ depending on the application requirements,
+As these differ depending on application requirements,
 only some basic implementation guidance is discussed in {{impl-consideration}}.
+This extension can be used with different scheduling algorithms that,
+e.g., can range from support for failover to simulatenous
+use of the aggreate capacity across all active paths.
+There are IETF specifications for path fail-over, but
```

Comment:
Do we really need to mention that there is a failover spec? As I said earlier I don’t think RFC7829 is necessarily relevant here as it doesn’t really assume that both paths are used simultaneously. If we want to keep this however I think we need to give a reference…?

```suggestion
```

Reply:
I agree that we should strike out that sentence. I was almost ready to commit the suggestion, but will let @mirjak do that once consensus is assessed.

I did read RFC7829, and I don't think citing it would be useful. The multipath extension does not create a new path fail-over capability, because that capability is already defined in RFC 9000. For example, the concern about simultaneous failover creating a thundering herd issue applies equally with and without the multipath extensions to RFC 9000.

Reply:
I hoped mentioning path-failover strengthened the position for making the QUIC MP Extension a PS. It did at least figure when MP items were chartered in the past. It is OK to remove this.

Reply:
I agree RFC7829 is not about concurrent usage of paths for data - I see this as opening two paths and can keep them alive until a path change is needed.